samples/simple-chat/csharp/FunctionApp/Functions.cs

```diff
@@ -61,8 +75,8 @@ public static class Functions
 
         [FunctionName("messages")]
         public static Task SendMessage(
-            [HttpTrigger(AuthorizationLevel.Anonymous, "post")]HttpRequest req,
-            [SignalR(HubName = "simplechat")]IAsyncCollector<SignalRMessage> signalRMessages)
+            [HttpTrigger(AuthorizationLevel.Anonymous, "post")] HttpRequest req,
```

Comment:
why change these lines

Reply:
automatically formatted

Reply:
ok..
codedflow doesn't show the space changes